docs/de/firefox_marketing.md

```diff
@@ -136,6 +140,7 @@ Bei Unsicherheiten kann auch die [Transvision Suche](https://transvision.mozfr.o
 |Internet health|Zustand des Internets|
 |Internet experience|Surf-Erlebnis|
 |JavaScript-Debugger|JavaScript-Debugger|
+|Join Firefox|Join Firefox<br>Komm zu Firefox(nicht auf CTA buttons nutzen, dort wird zwischen Sign in (einloggen) und Sign up (Anmelden oder Konto erstellen) differenziert.</br>|
```

Comment:
`Firefox(nicht` missing space

Reply:
Also `Join Firefox` should be removed from the German equivalent column, according to the doc

Reply:
Thank you @TheoChevalier for the review and spotting the error. She added more changes in the process. I will ask her about Join Firefox.